test/EndToEnd/ProjectBuildTests.cs

```diff
@@ -143,10 +143,12 @@ private void TestTemplateBuild(string templateName)
                 .Execute(restoreArgs)
                 .Should().Pass();
 
+            var dotnetRoot = Path.GetDirectoryName(RepoDirectoriesProvider.DotnetUnderTest);
             new BuildCommand()
-                .WithWorkingDirectory(projectDirectory)
-                .Execute()
-                .Should().Pass();
+                 .WithEnvironmentVariable("PATH", dotnetRoot) // override PATH since razor rely on PATH to find dotnet
```

Comment:
Other places that would need something like this?

Reply:
that's the only place with have rzc run underneath